CHANGELOG.md

```diff
@@ -1,12 +1,16 @@
 # Change log
 
+## dev
+
+* Fix overrides for test files
```

Comment:
what about changing it to `add scoped yola/ prefix for override rule`? I think its more informative

Reply:
Changelog must have description of things that is important to package user. The fix itself is not so important, though anyone who is interested can come and see commit of what exactly was broken.